xml/System.Text.Json.Serialization/JsonExtensionDataAttribute.xml

```diff
@@ -19,8 +19,24 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>When placed on a property of type <see cref="T:System.Collections.Generic.IDictionary`2" />, provides that any properties that do not have a matching member are added to that dictionary during deserialization and written during serialization.</summary>
```

Comment:
This sentence doesn't make sense. `... provides that any properties that ...` ???

```suggestion
    <summary>When placed on a property of type <see cref="T:System.Collections.Generic.IDictionary`2" />, any properties that do not have a matching member are added to that dictionary during deserialization and written during serialization.</summary>
```